examples/react-native/ios/Podfile

```diff
@@ -28,7 +28,7 @@ target 'ReactNative' do
   #
   # Note that if you have use_frameworks! enabled, Flipper will not work and
   # you should disable the next line.
-  use_flipper!()
+  # use_flipper!()
```

Comment:
```suggestion
  # NOTE: flipper disabled by default to improve iOS build times for CI
  # use_flipper!()
```

Reply:
Thanks, good call, will include this in my next PR, want to unblock CI